Library/Homebrew/cask/lib/hbc/artifact/symlinked.rb

```diff
@@ -39,7 +39,7 @@ def uninstall_phase
 
       def preflight_checks(source, target)
         if target.exist? && !self.class.islink?(target)
-          ohai "It seems there is already #{self.class.artifact_english_article} #{self.class.artifact_english_name} at '#{target}'; not linking."
+          opoo "It seems there is already #{self.class.artifact_english_article} #{self.class.artifact_english_name} at '#{target}'; not linking."
```

Comment:
I say we should fail. For all intents and purposes, the operation did not complete successfully, so it is an error.

Also think about a `binary`-only cask. If symlinking fails there, then the whole installation failed.